packages/react-components/src/components/LocalVideoTile.tsx

```diff
@@ -192,10 +195,35 @@ export const _LocalVideoTile = React.memo(
       showCameraSwitcherInLocalPreview
     ]);
 
-    const reactionOverlay =
-      reactionResources !== undefined ? (
-        <MeetingReactionOverlay overlayMode="grid-tiles" reaction={reaction} reactionResources={reactionResources} />
-      ) : undefined;
+    const videoTileOverlay = useMemo(() => {
+      const reactionOverlay =
+        reactionResources !== undefined ? (
+          <MeetingReactionOverlay overlayMode="grid-tiles" reaction={reaction} reactionResources={reactionResources} />
+        ) : undefined;
+      // If there are only one particiapnt in the call, show the waiting for others to join message
+      if (props.participantsCount === 1) {
+        return (
+          <>
+            {reactionOverlay}
+            <Stack
+              className={mergeStyles(videoContainerStyles, {
+                display: 'flex',
+                justifyContent: 'center',
+                alignItems: 'center',
+                backgroundColor: 'rgba(0, 0, 0, 0.6)'
+              })}
+            >
+              <p className={mergeStyles(waitingScreenIconStyle())}>&#9749;</p>
+              <Text className={mergeStyles(waitingScreenTextStyle(theme))} aria-live={'polite'}>
```

Comment:
Is this something we always want enabled or something we want contoso to have the ability to enable and disable?

Reply:
We always want it to be enabled